Makefile

```diff
@@ -26,5 +26,9 @@ verify-podnetworkconnectivitychecks:
 test-e2e-encryption: GO_TEST_PACKAGES :=./test/e2e-encryption/...
 .PHONY: test-e2e-encryption
 
-test-e2e-monitoring:
+test-e2e-monitoring: GO_TEST_PACKAGES :=./test/e2e-monitoring/...
+test-e2e-monitoring: GO_TEST_FLAGS += -v
+test-e2e-monitoring: GO_TEST_FLAGS += -timeout 5m
```

Comment:
https://github.com/openshift/library-go/pull/1823/files#r1804621250

Reply:
in a follow-up pr i would remove the timeout (let's use the default timeout) and the `-p` flag.